docs/user_guide/hopsworks/search.rst

```diff
@@ -0,0 +1,70 @@
+Search
+========================
+
+The search on the navigation bar can be used to search for Datasets, Feature groups, Features and Training datasets.
```

Comment:
Use the active voice, not the passive voice!
"You can search on the navigation bar for Datasets, Feature Groups, Features, and Training datasets.